components/map/overview.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Overview
+page_title: Map Overview
+description: Discover the Blazor Map and explore the examples.
+slug: components/map/overview
+tags: telerik,blazor,map,overview
+published: True
+position: 0
+---
+
+# Map Overview
+
+The Map displays geospatial information organized in layers. It is supported for both desktop and mobile devices.
+
+The component also provides tile layers, shape (vector) layers, and marker layers.
+
+## Basics
+
+To add a Telerik Map to your Blazor app:
+
+1. Add the `TelerikMap` tag.
+2. Add the `MapLayer` nested inside `MapLayers`.
+3. Provide the `Type` property.
+4. Set the `Attribution` and `Subdomains` properties.
+5. Provide the `UrlTemplate` property.
+
+The following example demonstrates how to set up the Map with its default configuration.
+
+>caption A basic configuration of the Telerik Map.
+
+````CSHTML
+@* This code snippet showcases an example of a basic Map configuration. *@
+
+<TelerikMap>
+    <MapLayers>
+        <MapLayer Type="@MapLayersType.Tile"
+                  Attribution="@Attribution"
+                  Subdomains="@Subdomains"
+                  UrlTemplate="@UrlTemplate">
+        </MapLayer>
+    </MapLayers>
+</TelerikMap>
+
+@code {
+    public string[] Subdomains { get; set; } = new string[] { "a", "b", "c" };
+    public string UrlTemplate { get; set; } = "https://#= subdomain #.tile.openstreetmap.org/#= zoom #/#= x #/#= y #.png";
+    public string Attribution { get; set; } = "&copy; <a href='https://osm.org/copyright'>OpenStreetMap contributors</a>";
+}
+````
+
+>caption The result from the above code snippet.
+
+![](images/map-basics.png)
+
+For more information, refer to the [article on layers]({%slug  components/map/layers%}) regarding the set of layers supported by the Map.
+
+## Features
+
+>caption The Map provides the following features:
+
+### General Properties
+
+* `TelerikMap.Center` - `double[]` - The map center. Coordinates are listed as [Latitude, Longitude].
+
+* `TelerikMap.MinZoom` - `double` - The minimum zoom level. Typical web maps use zoom levels from 0 (the whole world) to 19 (sub-meter features).
+
+* `TelerikMap.MaxZoom` - `double` - The maximum zoom level. Typical web maps use zoom levels from 0 (the whole world) to 19 (sub-meter features).
+
+* `TelerikMap.MinSize` - `double` - The size of the map in pixels at zoom level 0.
```

Comment:
All these settings can be replaced by a link to the API docs - their number is too large and the article becomes too long with an API docs-like structure.

What we can do instead is to add some sections regarding main features:
- Zoom
- Pan
- Navigator/Zoom controls
- Markers
- Layers

We do not need to go into too much detail, but this will give a general overview of the main functionalities, which is a lot more useful than listing a lot of API properties. The sections themselves can focus on 1-2 settings, so that the customers can make the connection.